src/server/app.js

```diff
@@ -0,0 +1,17 @@
+const express = require("express");
+const cors = require("cors");
+const app = express();
+const port = 3000;
+
+app.use(cors());
+app.use(express.json());
+
+app.post("/marcin", (request, response) => {
+  const { email } = request.body;
+  console.log("Odebrałem email:", email);
+  response.send("Email został odebrany!");
```

Comment:
we dont really have to send ("Email został odebrany!") we are not reading this string on frontend. the practice is to send back what has been added to a database, we are not adding to database but we can pretend.

response.send(200).(email)

Reply:
response.status(200).send(email); 
as GPT says: in send() we can put only sending content. Status ok, we should put in status()